### PR TITLE
Updated concepts/sdks/choose-authentication-provider

### PR DESCRIPTION
**Pull request by seesharprun**
I updated the documentation for this page based on the proposed updates to bring the **Microsoft.Graph.Auth** library forward to MSAL v3.

### CODE REVIEW

**Comment by seesharprun** [comment]
This PR is to fulfill a requirement of microsoftgraph/msgraph-sdk-dotnet-auth#39

**Comment by Lauragra** [comment]
@darrelmiller , please review this PR. Thanks.

File: concepts/sdks/choose-authentication-providers.md [hunk 1]
```diff
@@ -267,7 +289,11 @@ The interactive flow is used by mobile applications (Xamarin and UWP) and deskto
 # [C#](#tab/CS)
 
 ```csharp
-IPublicClientApplication clientApplication = InteractiveAuthenticationProvider.CreateClientApplication(clientId);
+IPublicClientApplication clientApplication = PublicClientApplicationBuilder
+            .Create(clientId)
+            .WithRedirectUri(redirectUri)
```
Comment (darrelmiller) [inline concepts/sdks/choose-authentication-providers.md]: redirectUri should not be necessary for InteractiveAuthenticationProvider.

**Comment by darrelmiller** [comment]
We need to synchronize the merge of this with the update and release of the auth library.

File: concepts/sdks/choose-authentication-providers.md (hunk 1 above)
Comment (seesharprun) [reply concepts/sdks/choose-authentication-providers.md]: I removed that line (294) in the latest commit

**Pull request merged**